misc/dev/dev.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+
```

Comment:
note for later: add a `--watch` option to automatically reload based on FS changes.